package.json

```diff
@@ -29,9 +29,11 @@
     "build-sample-reports": "yarn build-report && node build/build-sample-reports.js",
     "build-treemap": "node ./build/build-treemap.js",
     "build-viewer": "node ./build/build-viewer.js",
+    "build-types": "tsc --build tsconfig-all.json && rsync -a .tmp/tsbuildinfo/ ./ --include='*.d.ts' --include='*.d.cts' --exclude='*.map' --exclude='*.tsbuildinfo'",
     "reset-link": "(yarn unlink || true) && yarn link && yarn link lighthouse",
     "c8": "bash core/scripts/c8.sh",
     "clean": "rm -r dist proto/scripts/*.json proto/scripts/*_pb2.* proto/scripts/*_pb.* proto/scripts/__pycache__ proto/scripts/*.pyc *.report.html *.report.dom.html *.report.json *.devtoolslog.json *.trace.json shared/localization/locales/*.ctc.json || true",
+    "clean-types": "git clean -xf '*.d.ts' '*.d.cts' -e 'node_modules/' -e 'dist/' -e '.tmp/'",
```

Comment:
should this also exclude `types/`? Would be a bummer to clear out an in-progress file that's not committed yet

Reply:
In theory it should only include files that are gitignored, but not a bad idea